src/calculateRentalCost.js

```diff
@@ -28,7 +28,25 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let count = 0;
+  const priceday = 40;
+  const mincount = 20;
+  const maxcount = 50;
```

Comment:
use snake case for variables

```suggestion
  const carRentalPrice = 40;
  const minDiscount = 20;
  const maxDiscount = 50;
```